Sources/App/Settings/SettingsDetailViewController.swift

```diff
@@ -454,7 +453,26 @@ class SettingsDetailViewController: HAFormViewController, TypedRowControllerType
                         $0.disabled = true
                     },
                 ], getter: {
-                    Self.getSceneRows($0)
+                    var baseRows: [BaseRow] = []
+                    if $0 == scenes.first {
```

Comment:
i think it would be better to put a new section at the top for this, or a new RealmSection watching it. this is sort of side-effect-y -- this 'value' needs to update live with contents other than the first one coming in/out, too, right?

Reply:
Good one, I forgot to reflect the state on manually changing the scene toggles. It's done now in my latest commit.

Reply:
![Uploading Simulator Screen Recording - iPhone 15 Pro - 2023-11-06 at 09.38.51.gif…]()